meson.build

```diff
@@ -39,13 +39,15 @@ add_project_arguments(
   '-I' + meson.build_root(),
   language: 'cpp', native: true)
 
+# NOTE: -std=gnu11 is required to enable inline assembly.
 add_project_arguments(
-  '-std=gnu99', '-Wall', '-Werror',
+  '-std=gnu11', '-Wall', '-Werror',
   '-I' + meson.source_root(),
   '-I' + meson.build_root(),
+  '-isystem' + meson.source_root() / 'sw/device/freestanding',
   language: 'c', native: false)
 add_project_arguments(
-  '-std=gnu99', '-Wall', '-Werror',
+  '-std=gnu11', '-Wall', '-Werror',
```

Comment:
#1108 using `-std=c11` and uses a macro to cover `asm`/`__asm__`.  There isn't any inline assembly in this change though, is this still needed?

Reply:
#1108 is pending; once that's submitted I'll switch this over to c11 (this is really just so that we get passing CI for now).

Reply:
Yeah there's a fun dependency between those two patches. Whichever lands second will need to be updated, but that's easy.